content/millennium/r4/medications/immunization.md

```diff
@@ -166,6 +166,58 @@ The `ETag` response header indicates the current `If-Match` version to use on su
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Update
+
+Update an Immunization.
+
+    PUT /Immunization/:id
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true, patient: false) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/json+fhir',
+                   'Content-Type': 'application/json+fhir', 'If-Match': 'W/"&lt;Current version of the Immunization resource>"'} %>
```

Comment:
can remove Accept, and switch content-type to application/fhir+json

Reply:
Updated. https://github.com/cerner/fhir.cerner.com/pull/295/commits/fe9f32d2c48645c5ba462f17793818a3e32b5d4a#diff-3e96067fd99a2fee85880712d23a6719R181